sandbox-deploy/create_service_account.sh

```diff
@@ -52,41 +49,35 @@ while getopts ":hms:u:r:o:" opt; do
     o)
       org=${OPTARG}
       ;;
-    m)
-      org_manager_output="-m"
-      org_manager="true"
-      ;;
     h)
       echo "$usage"
       exit 0
       ;;
   esac
 done
 
-if [[ $space = "" || $service = "" ]]; then
+if [[ -z "$space" || -z "$service" ]]; then
   echo "$usage" >&2
   exit 1
 fi
 
-cf target -o $org -s $space >&2
+cf target -o "$org" -s "$space" >&2
```

Comment:
"I like it!"

Reply:
your previous reviews were in my head as I made those changes